File: test/ocaml/fib-iterative.ml
let fib n =
   let rec fib' m a b =
      match m with
          0 -> b
        | _ -> fib' (m - 1) b (a + b)
   in
   fib' n 0 1;;

let t0 = Sys.time() in
let ans = (fib (int_of_string (Sys.argv.(1)))) in
let time = Sys.time() -. t0 in
Format.printf "%d, %fs\n" ans time

File: test/js/fib-iterative.js
const fib = n => {
   let a = 0,
      b = 1
   for (let i = 0; i < n; i++) [a, b] = [b, a + b]
   return b
}

const t0 = process.hrtime()
fib(eval(process.argv[2]))
console.log(process.hrtime(t0))
